lib/fastlane/plugin/jira_release_notes/actions/jira_release_notes_action.rb

```diff
@@ -106,6 +111,12 @@ def self.available_options
                                        description: "Jira issue status",
                                        sensitive: true,
                                        default_value: ""),
+          FastlaneCore::ConfigItem.new(key: :components,
+                                       env_name: "FL_JIRA_components",
```

Comment:
environment variables are named in uppercase

Reply:
done here 0e08e8d